scripts/main.js

```diff
@@ -49,6 +51,14 @@ Hooks.once('ready', async () => {
 
     await settings.setCustomEffectsItemId(item.id);
   }
+
+  /**
+   * Migrate the effect descriptions from legacy CEs to the description field of Active Effects and clear up flags.
+   */
+  await migrations._migrateCustomEffectsItemEffectDescriptions(
+    settings.customEffectsItemId,

```

Comment:
You don't need to pass this in since you have the settings in the migrations file

Reply:
Actually, probably just don't pass anything in at all.

Reply:
And also this could turn into the `execute` function mentioned in another comment